tests/unit/angularTokenAuth-spec.js

```diff
@@ -9,18 +9,6 @@
         // Load the generic module.
         beforeEach(module('angular-token-auth'));
 
-        beforeEach(function () {
-
-            // Inject dependencies. module() can no longer be called after this.
-            inject(function ($rootScope) {
-                // Inject something to create the injector.
-                // Uncomment this line if you are having test troubles. The
-                // injector won't run if there are missing dependencies.
-                // console.log('generic: If this log does not show, there are missing dependencies.');
-            });
```

Comment:
This is required for `it('should load the dependencies without error')` to run below. No specs will run without an injector being created.